src/content/docs/blog/2025/06/30/mid-year-update/index.md

```diff
@@ -0,0 +1,185 @@
+
+To ease the transition to the new repository for existing users, we are working on a script that can automatically modify the active repository on the system.
+
+Once this script has been sufficiently productized, the next time existing users update their systems, they will notice that every single package will show an update available. 
+
+The exact number will vary from system to system depending on how many other packages are installed from the repository but for context, on a base AerynOS GNOME install, this is around 500 packages.
+
+In the meantime, we have created a manual guide on how to transition existing installs to the new repository in our GitHub Discussions forum [here](https://github.com/orgs/AerynOS/discussions/53). The process is fairly simple, but if you do have any issues transitioning manually, do get in touch via a GitHub Discussions post or via [Matrix](https://matrix.to/#/#aerynos:matrix.org).
+
+### Content Delivery Network for Packages and ISOs
+
+A common bit of feedback we have been receiving relates to the download speed of our repository, namely that it is not fast or even acceptable, especially if you live outside of Europe. This became more evident for those using the rebuild repository on ermo's rebuild testing server, which felt noticeably faster for people in Europe in particular.
+
+To remedy this, we have implemented CDN caching for our new `cdn.aerynos.dev` hosted assets. This means there will be synced copies of our ISOs and package repository on CDN servers around the world, which should help improve download speeds.
+
+In particular, the new rolling `unstable` package repository mentioned above will be served via this CDN for the benefit of our users.
+
+Please let us know how you get on with AerynOS ISO and package downloads in the coming weeks, as we would love to validate the improvement outside of our own internal testing.
+
+## Future infra development targets
+
+So far, we have only outlined what we have already accomplished since late March.
+
+The next part of this blog post is going to be a brief outline of where we are going from here in terms of infra and repository development.
+
+### Versioned Repositories
+
+With the transition to the new infra and the new `unstable` repository, we have been freed up to begin planning out the necessary steps to be able to deliver versioned repositories and versioned moss format upgrades.
+
+These topics have been mentioned in a previous blog [post](https://aerynos.com/blog/2025/02/06/hello-2025/#-versioned-repositories).
+
+### How do versioned repositories add value?
+
+Versioned Repositories will enable us to deploy new boulder and moss features in a seamless fashion. This will enable us to introduce breaking code and on-disk format changes, that would otherwise cause installed systems to require manual intervention for them to continue to receive updates.
```

Comment:
Hmm, so here "Versioned Repositories" is written like a proper noun again. Is it supposed to be?